code/__DEFINES/dcs/signals.dm

```diff
@@ -830,6 +830,16 @@
 ///from monkey CtrlClickOn(): (/mob)
 #define COMSIG_XENO_MONKEY_CLICK_CTRL "xeno_monkey_click_ctrl"
 
+// /datum/component/rot component
+///to rot start_processing()
+#define COMSIG_START_MIASMA_GENERATION "rot_start"
+///to rot stop_processing()
+#define COMSIG_STOP_MIASMA_GENERATION "rot_stop"
```

Comment:
Where is this signal being used? I seem not to be finding where it's sent.

Reply:
Check the rot component

Reply:
I see it being registered to, but never sent.

Reply:
I don't remember why I added it, I don't think I've got a use for it so I'm removing it.